Automattic/delphin
# My Domains: Add new illustrations

Adds new illustrations to My Domain's cards.

Also updates the nameserver card per this comment: https://github.com/Automattic/delphin/issues/939#issuecomment-261956888

**Before** | **After**
----------- | -----------
![image](https://cloud.githubusercontent.com/assets/12596797/20573728/3ee60346-b17f-11e6-9b25-c19c7a4dd01b.png) | ![image](https://cloud.githubusercontent.com/assets/12596797/20573644/d9a9d6ce-b17e-11e6-8be2-8efa9c383bca.png)

**Testing**
1. Go to My Domains.
2. Make sure you have a domain in each of the 4 states.
3. Verify that the illustrations look ok at all screen sizes.
